src/AppBundle/Controller/EventManagement/EventController.php

```diff
@@ -107,23 +141,431 @@ public function addEvent(Request $request)
+                                )
+                            );
+                        }
+                        break;
+                    case "comment":
+                        if ($participationFields[$i][2]) {
+                            $form->add(
+                                'comment',
+                                TextareaType::class,
+                                array(
+                                    "attr" => ["placeholder" => "general.comment"],
+                                    'label' => "general.comment",
+                                    'empty_data' => '',
+                                    "required" => $participationFields[$i][3]
+                                )
+                            );
+                        }
+                        break;
+                }
+            }
+            $form = $form->add('save', SubmitType::class, array(
+                    "label" => "Event.attendInvitationLink.submit",
+                    "attr" => ["class" => "btn btn-lg btn-primary btn-block"]))
+                ->getForm();
+
+            $form->handleRequest($request);
+
+            /* handle submitted form */
+            if ($form->isSubmitted() && $form->isValid() && $valid_now) {
+                /* proof Google reCaptcha */
+                $reCaptchaSecret = $this->container->getParameter('recaptcha.secret');
+                if ($this->get("reCaptcha")->validateReCaptcha($request->request->get("g-recaptcha-response"), $reCaptchaSecret)) {
+                    /* save input */
+                    $eventAttend = $form->getData();
+
+                    $em->persist($eventAttend);
+                    $em->flush();
+
+                    $this->addFlash("success", "Vielen Dank, Ihre Anmeldung ist bei uns erfolgreich eingegangen.");
+
+                    return $this->redirectToRoute('login');
+                }
+                $this->addFlash("error", "Bestätige bitte den Spamschutz!");
+            }
+
+            return $this->render('eventManagement/attendInvitationLink.html.twig', array(
+                "loggedInUser_firstname"=>$loggedInUser_firstname,
+                "loggedInUser_lastname"=>$loggedInUser_lastname,
+                "loggedInUser_Stamm"=>$loggedInUser_Stamm,
+                "event"=>$event,
+                "registrationAttendInvitationLink"=>$form->createView(),
+                "showParticipationFields"=>$this->FieldsToShowInTWIG($event->getParticipationFields()),
+                "valid_now"=>$valid_now,
+            ));
+        } else {
+            $this->addFlash("error", "Dieser Einladungslink ist leider nicht mehr gültig!");
+            return $this->redirectToRoute("login");
+        }
+    }
+
+    /**
+     * @param int $length
+     * @param string $characters
+     * @return string
+     */
+    private function generateRandomString($length = 16, $characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
```

Comment:
Wollen wir das in eine Utile Classe tun? Oder gibt es das eventuells chon von PHP?